cpp/ql/src/semmle/code/cpp/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -140,6 +141,8 @@ predicate compatibleTypes(Type t1, Type t2) {
   any() // stub implementation
 }
 
+private predicate suppressUnusedThis(Type t) { any() }
```

Comment:
Sorry, I should have clarified: Here, we aren't using it for a `this` variable, so it ought to be `suppressUnusedType`.

Reply:
Ah, right. Fixed.